contracts/core/src/contract.rs

```diff
@@ -739,16 +734,17 @@ fn execute_tick_idle(
                     attrs.push(attr("knot", "022"));
                     attrs.push(attr("state", "staking_rewards"));
                 } else {
-                    attrs.push(attr("knot", "024"));
-                    if let Some(unbond_message) =
-                        get_unbonding_msg(deps.branch(), &env, config, &info)?
-                    {
+                    let (unbond_message_relut, unbond_attrs) =
+                        get_unbonding_msg(deps.branch(), &env, config, &info)?;
+                    if let Some(unbond_message) = unbond_message_relut {
```

Comment:
```suggestion
                    let (unbond_message_result, unbond_attrs) =
                        get_unbonding_msg(deps.branch(), &env, config, &info)?;
                    if let Some(unbond_message) = unbond_message_result {
```

Reply:
Rewrote this code to pass by reference instead of returning it, looks cleaner